### DISCUSSION TITLE
Please Some one kindly help me

### DISCUSSION THREAD

**Discussion by debidattasuryaprakash** [appwrite/appwrite#4301] [General]
I have added the collection ID and added the role to the users in the settings of the database collection

![Screenshot from 2022-10-05 23-37-26](https://user-images.githubusercontent.com/74270500/194131837-aa6cd1b0-2299-487d-9292-741e6f7bf786.png)

But each time adding a todo gives me this error

![Screenshot from 2022-10-05 23-38-05](https://user-images.githubusercontent.com/74270500/194131899-13d7a8c8-cb3c-43d5-9cca-5e5d46099b99.png)

**Comment by GHolanda14** [comment]
Could you show us your source code, so we can understand a little bit better what this issue is about?

**Comment by stnguyen90** [comment]
> Missing required parameter: "data"

This means you're passing in nothing for data which isn't allowed.
